CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+* The **lint** command will not use network by default when running unit-tests anymore.
```

Comment:
Lets leave a link to the docs

Reply:
here?

Reply:
yeah disregard this. not the right place to put it